web/vite.config.js

```diff
@@ -4,4 +4,10 @@ import react from '@vitejs/plugin-react'
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [react()],
+  server: {
```

Comment:
I am still facing the same problem, and I'm pretty sure it's because Vite is only listening on localhost inside the container. The logs confirm this - they only show:  `Local: http://localhost:5173/` and  `Network: use --host to expose`.

Since host: '0.0.0.0' is missing in the vite.config.ts, the server is refusing the connection coming from Windows.

```suggestion
  server: {
    host: '0.0.0.0',
```
Adding host: 0.0.0.0 should do the trick.

Reply:
other fixes look good btw 👍